brain-marks/Localization/en.lproj/Localizable.strings

```diff
@@ -42,7 +42,7 @@
 
 // SettingsView
 "Settings" = "Settings";
-"BrainMarksIsOpenSource..." = "Brain-marks is an open source project! It was created for the Big Brain Hackathon and during Hacktoberfest 2021, work has continued.";
+"BrainMarksIsOpenSource" = "Brain-marks is an open source project! It was created for the Big Brain Hackathon and during Hacktoberfest 2021 and 2022, work has continued.";
```

Comment:
Can you use the exact text that I noted in #133 ? I didn't want to include the year anymore, and the hyphen in "brain marks" is gone at the beginning

Reply:
Changed it accordingly to the issue